contenthub-editor.php

```diff
@@ -1,7 +1,7 @@
 <?php
 /**
  * Plugin Name: ContentHub Editor
- * Version: 5.16.0
+ * Version: 5.17.0
```

Comment:
Ikke for at være helt på tværs, men det er nok mere en bugfix, end en feature addition ;-)
```suggestion
 * Version: 5.16.1
```